src/components/Match/Match.css

```diff
@@ -148,6 +159,117 @@
   vertical-align: middle;
 }
 
+.ward-log {
+  width: 100%;
+  font-size: .8em;
```

Comment:
We have some defined font sizes in palette.css. I guess 0.8em is somehing like 12px so u can use `--fontSizeSmall`